debian/control

```diff
@@ -21,7 +21,7 @@ Conflicts: mediabrowser, emby, emby-server-beta, jellyfin-dev, emby-server
 Architecture: any
 Depends: at,
          libsqlite3-0,
-         jellyfin-ffmpeg (>= 4.2.1-2),
+         jellyfin-ffmpeg (>= 4.3.1-1),
```

Comment:
OK, we can't change this. Because we can't build the new FFmpeg for Stretch due to dav1d. And I don't want to futz around with that too much, so we can leave Stretch installs on the old version of jellyfin-ffmpeg while Buster uses the new one. I don't believe this causes any long-term conflicts, yes? Aside from fixing bugs?